docs/spelling_wordlist.txt

```diff
@@ -0,0 +1,1366 @@
+MessageAttributes
+Metastore
+Metatsore
+Mixin
+Mongo
+Moto
+Mysql
+NFS
+NaN
+Naik
+Namenode
+Namespace
+Nextdoor
+Nones
+NotFound
+Nullable
+Nurmamat
+OAuth
+Oauth
+Oauthlib
+Okta
+Oozie
+Opsgenie
+Optimise
+PEM
+PTarget
+Pagerduty
+Papermill
+Parallelize
+Parameterizing
+Paramiko
+Params
+Pem
+Pinot
+Popen
+Postgres
+Postgresql
+Potiuk
+Pre
+Precommit
+Preprocessed
+Proc
+Protobuf
+PublisherClient
+Pubsub
+Py
+PyPI
+Pylint
+Pyspark
+PythonOperator
+Qingping
+Qplum
+Quantopian
+Qubole
+Quboles
+Quoble
+RBAC
+Readme
+Realtime
+Rebasing
+Rebrand
+Reddit
+Redhat
+Reinitialising
+Retrives
+Riccomini
+Roadmap
+Robinhood
+SIGTERM
+SSHClient
+SSHTunnelForwarder
+SaaS
+Sagemaker
+Sasl
+SecretManagerClient
+Seedlist
+Seki
+Sendgrid
+Siddharth
+SlackHook
+SparkPi
+SparkR
+SparkSQL
+Sql
+Sqlalchemy
+Sqlite
+Sqoop
+Stackdriver
+Standarization
+StatsD
+Statsd
+Stringified
+Subclasses
+Subdirectory
+Submodules
+Subpackages
+Subpath
+SubscriberClient
+Subtasks
+Sumit
+Swtch
+Systemd
+TCP
+TLS
+TTY
+TThe
+TZ
+TaskInstance
+Taskfail
+Templated
+Templating
+Teradata
+TextToSpeechClient
+Tez
+Thinknear
+ToC
+Tomasz
+Tooltip
+Tsai
+UA
+UNload
+Uellendall
+Umask
+Un
+Undead
+Undeads
+Unpausing
+Unpickle
+Upsert
+Upsight
+Uptime
+Urbaszek
+Url
+Utils
+Vendorize
+Vertica
+Vevo
+WTF
+WaiterModel
+Wasb
+WebClient
+Webhook
+Webserver
+Werkzeug
+Wiedmer
+XCom
+XComs
+Xcom
+Xero
+Xiaodong
+Yamllint
+Yandex
+Yieldr
+Zego
+Zendesk
+Zhong
+Zsh
+Zymergen
+aIRFLOW
+abc
+accessor
+accountmaking
+acessible
+ack
+ackIds
+acknowledgement
+actionCard
+acurate
+acyclic
+adhoc
+aijamalnk
+airbnb
+airfl
+airflowignore
+ajax
+alertPolicies
+alexvanboxel
+allAuthenticatedUsers
+allUsers
+allowinsert
+analyse
+analytics
+analyticsreporting
+analyzeEntities
+analyzeSentiment
+analyzeSyntax
+aoen
+apache
+api
+apikey
+apis
+appbuilder
+approle
+arg
+args
+arn
+arraysize
+artwr
+asc
+ascii
+asciiart
+asia
+assertEqualIgnoreMultipleSpaces
+ast
+async
+athena
+attemping
+attr
+attrs
+auth
+authMechanism
+authenication
+authenticator
+authorised
+autoclass
+autocommit
+autocomplete
+autodetect
+autodetected
+autoenv
+autogenerated
+autorestart
+autoscale
+autoscaling
+avro
+aws
+awsbatch
+backend
+backends
+backfill
+backfilled
+backfilling
+backfills
+backoff
+backport
+backreference
+backtick
+backticks
+balancer
+baseOperator
+basedn
+basestring
+basetaskrunner
+bashrc
+basph
+batchGet
+bc
+bcc
+beeen
+behaviour
+behaviours
+behavoiur
+bicket
+bigquery
+bigtable
+bitshift
+bolkedebruin
+booktabs
+boolean
+booleans
+bootDiskType
+boto
+botocore
+bowser
+bq
+bugfix
+bugfixes
+buildType
+bytestring
+cacert
+callables
+cancelled
+carbonite
+cas
+cassandra
+casted
+catchup
+cattrs
+ccache
+celeryd
+celltags
+cfg
+cgi
+cgroups
+changelog
+charset
+checklicence
+chmod
+ci
+cid
+ciphertext
+classable
+classname
+classpath
+classpaths
+cli
+clientId
+cloudml
+cloudp
+cloudsqldatabehook
+cloudwatch
+cmd
+cmdline
+cmds
+cname
+cnf
+codebase
+codecov
+colour
+colours
+commiter
+comnand
+compaitible
```

Comment:
This all are spelling errors

Reply:
`commiter` is used in the string `<li><p>[AIRFLOW-XXXX] Fix typo commiter =&gt; committer (#7392)</p></li>`. thoughts on how to handle that?